src/main/java/uk/gov/hmcts/reform/divorce/validationservice/domain/request/ValidationRequest.java

```diff
@@ -25,6 +26,6 @@
 
     @JsonProperty(value = "data", required = true)
     @ApiModelProperty(value = "data to be validated", required = true)
-    private final CoreCaseData data;
+    private final Object data;
```

Comment:
Is there a need to be an Object type and not a specific type, and if required can we make use of generics ?.

Reply:
Request to make as techdebt.
Object works for now, making it a generic type is possible but not trivial at this point in time.